flight/config.py

```diff
@@ -0,0 +1,26 @@
+"""File to hold important constant values and configure drone upon startup"""
+from mavsdk import System
+
+WAIT: float = 2.0  # Seconds
+
+
+async def config_params(drone: System) -> None:
+    """
+    Sets certain parameters within the drone for flight
+
+    Parameters
+    ----------
+        drone: System
+            MAVSDK object for manual drone control & manipulation
```

Comment:
Don't need to indent this past parameters
```suggestion
    Parameters
    ----------
    drone: System
        MAVSDK object for manual drone control & manipulation
```